pkg/operator/target_config_reconciler_v311_00.go

```diff
@@ -157,6 +172,34 @@ func managePod_v311_00_to_latest(client coreclientv1.ConfigMapsGetter, recorder
 	return resourceapply.ApplyConfigMap(client, recorder, configMap)
 }
 
+func checkForFeatureGates(featureGateLister configlistersv1.FeatureGateLister) (map[string]bool, error) {
+	featureGateListConfig, err := featureGateLister.Get("cluster")
+	var featureGates = make(map[string]bool)
+	if err != nil {
```

Comment:
Need to pull in the default FeatureSet if the cluster FeatureGate cannot be found.

Reply:
Done. PTAL.